tests/e2e/lib/blocks/mailchimp.js

```diff
@@ -40,7 +40,7 @@ export default class MailchimpBlock {
 		global.page = loginTab;
 
 		if ( ! isLoggedIn ) {
-			await ( await LoginPage.init( loginTab ) ).login( 'defaultUser' );
+			await ( await LoginPage.init( loginTab ) ).login( 'defaultUser2' );
```

Comment:
I'd rather us try to clean out the defaultUser (and have a regular job to do so) instead of switching over to the second one. We'd end up with the same problem.

Reply:
Agreed. I reverted this change